finance.adoc

```diff
@@ -124,7 +124,7 @@ A cash advance is a type of financing that is offered to merchants based on thei
 |INVOICE_PAYMENT |References an invoice payment.
 |INVOICE_PAYMENT_FEE |References an invoice payment fee.
 |PAYMENT |References an alternative, third-party, payment method where iZettle handles the funds e.g. PayPal QR code(only DE and FR), Klarna QR code(only SE, FI, DK and NO).
-|PAYMENT_FEE |References the fee for a third-party payment method e.g PayPal QR code(only DE and FR), Klarna QR code(only SE, FI and DK).
+|PAYMENT_FEE |References the fee for a third-party payment method e.g PayPal QR code(only DE and FR), Klarna QR code(only SE, FI, DK and NO).
```

Comment:
It's a bit maintenance-heavy to explicitly write out the markets these products currently are available in, and will shortly be outdated. Consider all the different regional differences we have across all our products.

 What do you think about simply not mentioning this at all in the documentation?